src/textual/tree-sitter/highlights/html.scm

```diff
@@ -1,64 +1,13 @@
 (tag_name) @tag
-(erroneous_end_tag_name) @html.end_tag_error
+(erroneous_end_tag_name) @tag.error
```

Comment:
We'll have to be mindful about changing existing highlight queries as it'll break features of some themes. For example, `@html.end_tag_error` is used in a couple of builtin themes: 

https://github.com/Textualize/textual/blob/5dfa644dfabde1fdcb59024ff9b18cdc59d92014/src/textual/_text_area_theme.py#L217

Reply:
Note that the `@html.end_tag_error` is completely arbitrary (a choice by us), so we could call it anything we want. However, if anyone has created their own theme it would no longer map correctly.